Tema3/src/Exercitiul2.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+
+/**
+ * Fahrenheit to Celsius Conversion
+ * create a method that converts temperature from Fahrenheit to Celsius.
+ * write a program that for a given (by the user) body temperature, measured
+ * in Fahrenheit degrees, outputs it in Celsius degrees, with the following message:
+ * "Your body temperature in Celsius degrees is X", where X is respectively the Celsius degrees.
+ * In addition if the measured temperature in Celsius is higher than 37 degrees, the program
+ * should warn the user that they are ill, with the following message "You are ill!".
+ *
+ * @author Bogdan
+ */
+
+public class Exercitiul2 {
+    public static void main(String arg[]) {
```

Comment:
The homework was to create a method that does the conversion, so please extract the functionality that does the conversion to a method and call that method when needed